lib/peep.rb

```diff
@@ -0,0 +1,17 @@
+require 'pg'
+
+class Peep
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_chatter_test')
+    else
+      connection = PG.connect(dbname: 'chitter_chatter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps;")
+    peeps = result.map { |peep| peep['content'] }
+    peeps.reverse
```

Comment:
If I want the results to be in chronological order, I think it might be better to sort by timestamp with SQL query:
SELECT * FROM peeps ORDER BY (timestamp column name) DESC;